cypress-tests/cypress/support/pageBuilder/pbDeleteAllMenus.ts

```diff
@@ -0,0 +1,82 @@
+import { gqlClient } from "../utils";
+
+const LIST_MENUS_QUERY = /* GraphQL */ `
+    query listMenus {
+        pageBuilder {
+            listMenus {
+                data {
+                    title
+                    slug
+                    description
+                    items
+                    createdOn
+                    createdBy {
+                        id
+                        displayName
+                    }
+                }
+            }
+        }
+    }
+`;
+
+const DELETE_MENU_MUTATION = /* GraphQL */ `
+    mutation deleteMenu($slug: String!) {
+        pageBuilder {
+            deleteMenu(slug: $slug) {
+                error {
+                    code
+                    message
+                }
+            }
+        }
+    }
+`;
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            listMenus(): Promise<Record<string, any>[]>;
+            deleteMenu(slug: string): Promise<Record<string, any>>;
+            pbDeleteAllMenus(): void;
+        }
+    }
+}
+
+Cypress.Commands.add("listMenus", () => {
```

Comment:
If we want to have these commands potentially for later use (they are not being used just in this file), let's extract these new commands into separate files, the same way we're doing with other commands.

P.S. Don't forget the `ps` prefix, for example `pbListMenus`.

Reply:
I have separated them into pbListMenus and pbDeleteMenu with pbDeleteAllMenus remaining as well.

Sadly, none of it works anymore when I separate them and i'm really not sure why. I tried debugging cy.pbListPages but it always returns a null. We can check it out in our next call if we get the time.